### DISCUSSION TITLE
UPDATE question

### DISCUSSION THREAD

**Discussion by ondohotola** [mithrandie/csvq#116] [Q&A]
Please see this [QSV discussion](https://github.com/jqnatividad/qsv/discussions/1270) and if there is a way of doing this much faster better in CSVQ, please post here or there.

**Comment by mithrandie** [comment]
This query is more faster.

```sql
UPDATE i
   SET i.eppstatus = e.eppstatus
  FROM DomainsInclude i
       JOIN DomainsExclude e
         ON e.name = i.name
```

However, the speed is not very fast and csvq may not give you the speed you need.

**Comment by ondohotola** [comment]
That's fast enough for my purposes.

Thanks

**Comment by ondohotola** [comment]
Option 1 of this [QSV discussion](https://github.com/jqnatividad/qsv/discussions/1270) is even faster.

Thank you again.
